lib/compiler/passes/generate-js.js

```diff
@@ -3,7 +3,7 @@
 const asts = require("../asts");
 const js = require("../js");
 const op = require("../opcodes");
-const VERSION = require('../../version')
+const VERSION = require("../../version");
```

Comment:
unrelated lint bug that needs to be landed even if this patch doesn't.

Reply:
I think that this should be in the it's own PR or, at least, commit, together with other two unrelated changes in the `package.json`